Misc/NEWS.d/next/Core and Builtins/2020-03-28-14-28-46.bpo-40077.4C_xJm.rst

```diff
@@ -0,0 +1,3 @@
+Replace statically allocated types with heap allocated types by using
+:c:func:`PyType_FromSpec`. Add a module state to store the _abc_data_typ Add
+traverse, clear and free functions to the module.
```

Comment:
Please remove the NEWS entry.

The modified type is not accessible not visible by end users. I don't think that it's worth it to document such enhancement for "end users".